citus_metadata_sync.c

```diff
@@ -0,0 +1,43 @@
+/*-------------------------------------------------------------------------
+ *
+ * citus_metadata_sync.c
+ *
+ * This file contains functions to sync pg_shard metadata to the CitusDB
+ * metadata tables.
+ *
+ * Copyright (c) 2014, Citus Data, Inc.
+ *
+ *-------------------------------------------------------------------------
+ */
+
+#include "postgres.h"
+
+#include "citus_metadata_sync.h"
+#include "distribution_metadata.h"
+
+#include "utils/builtins.h"
+
+
+/* declarations for dynamic loading */
+PG_FUNCTION_INFO_V1(partition_column_to_node_string);
+Datum
```

Comment:
Could you add spacing between the declaration, and also more importantly add a function comment?